### DISCUSSION TITLE
Package for nixos

### DISCUSSION THREAD

**Discussion by Dilly-hue** [TheAssassin/AppImageLauncher#670] [Ideas]
### Pre-submit checks

- [X] I checked for similar issues beforehand, but could not find any. I could not add my proposal to any existing issue.
- [X] I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise.

### Feature description

I would like to have App Image launcher on my nixos system build.
There is still advantages to have app images on a nixos due to their portability

### Proposed solution

Package AppImageLaucher so that it could be installed on NixOS

**Comment by TheAssassin** [comment]
You should talk to NixOS instead. I do not intend to provide end user packaging for (niche) distributions, especially since I'd have to learn packaging first.
